src/modules/dream/geometry/DreamGeometry.cpp

```diff
@@ -43,8 +43,8 @@ int DreamGeometry::getPixel(Config::ModuleParms &Parms,
     break;
   }
 
-  if (Pixel == 0) {
-    XTRACE(DATA, WAR, "Invalid pixel returned");
+  if (Pixel < 1) {
+    XTRACE(DATA, WAR, "Invalid pixel returned: %d", Pixel);
```

Comment:
In princiuple I am ready to approve ths change. But I don't like the change in logic.
The check should be == 0 which is the definition of a bad pixel. Anything returning -1 should 
crash the application instead :-)